_action_files/pr_comment.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
can we leave this with `/sh`. there is a reason for this

Reply:
Sure! Can you elaborate on this a bit please?